apps/api/src/app/shared/dtos/pagination-response.ts

```diff
@@ -8,8 +8,9 @@ export class PaginatedResponseDto<T> {
 
   @ApiProperty({
     description: 'Total count of items matching the query',
+    required: false,
   })
-  totalCount: number;
+  totalCount?: number;
```

Comment:
What's the reason behind making it optional? Being an interface for paginated responses I do think that `totalCount` should be mandatory. Same for `skip` and `limit`.
I can see that the use case modified has a different type of response that includes `hasMore`.  Not sure when it was introduced but we might need to create a different type for `PaginatedResponseWithHasMore` if they have to share some of the functionalities.
Am I missing anything from the context?